interface/i18n/mist.de.i18n.json

```diff
@@ -48,7 +48,7 @@
                 "mainNetwork": "Hauptnetzwerk",
                 "startMining": "⛏ Mining starten (nur auf Testnetz)",
                 "stopMining": "⛏ Mining stoppen",
-                "externalNode": "using external node",
+                "externalNode": "externer Node aktiv",
```

Comment:
As there is no consens about the grammatical sex, we can choose according to [these rules](http://www.vds-ev.de/fallbeispiele/945-artikelwahl-bei-englischen-substantiven):
- semantisches Prinzip (der Knoten -> der Node)
- phonologisches Prinzip (die Not -> die Node)
- graphemisches Prinzip (die Mode -> die Node)

I would prefer to go with the semantic principle (see https://de.wikipedia.org/wiki/Node)